addons/website/static/src/components/configurator/configurator.xml

```diff
@@ -54,9 +54,9 @@
                     </div>
                     <span t-att-class="!state.selectedType ? 'o_configurator_hide' : 'o_configurator_show'"> for my</span>
                 </div>
-                <div t-attf-class="o_configurator_typing_text d-inline d-md-block o_configurator_industry mb-md-2 mb-lg-4 {{!state.selectedType ? 'o_configurator_hide' : 'o_configurator_show'}}">
-                    <div class="o_configurator_industry_wrapper position-relative d-inline">
-                        <i t-attf-class="industry_selection d-inline d-md-inline-block rounded bg-100 px-2 px-md-3 mx-2 mx-md-0 {{state.selectedIndustry ? 'o_step_completed' : 'o_step_todo show'}}" contenteditable="True" t-on-blur="blurIndustrySelection" t-on-input="inputIndustrySelection" t-ref="industrySelection"/>
+                <div t-attf-class="o_configurator_typing_text d-inline d-md-flex align-items-start o_configurator_industry mb-md-2 mb-lg-4 {{!state.selectedType ? 'o_configurator_hide' : 'o_configurator_show'}}">
+                    <div class="o_configurator_industry_wrapper position-relative d-inline mr-2">
+                        <i t-attf-class="industry_selection overflow-hidden d-inline d-md-inline-block rounded bg-100 px-2 px-md-3 mx-2 mx-md-0 {{state.selectedIndustry ? 'o_step_completed' : 'o_step_todo show'}}" contenteditable="True" t-on-blur="blurIndustrySelection" t-ref="industrySelection"/>
                     </div>
                     <span> business</span>
```

Comment:
Unrelated to the PR but I discover this code slowly but surely: this feature will never be translatable ever this way...

In french: "Je veux un blog pour mon club de tennis entreprise"... In other languages, this will be worse :/ I don't think there is a good solution without making 3 sentences a clearly not cutting them in multiple parts :grimacing:

Reply:
@sbuhl just to make you aware of this tricky problem. We have no straightforward solution for now...